From: Andrzej Bieniek <andyhelp@gmail.com>
Date: Sun, 11 Sep 2011 21:10:47 +0100
Subject: Fixed --version option.

---
 GstDebugViewer/Main.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/GstDebugViewer/Main.py b/GstDebugViewer/Main.py
index 8be8e4b..b401ccd 100644
--- a/GstDebugViewer/Main.py
+++ b/GstDebugViewer/Main.py
@@ -27,7 +27,7 @@ Common = GstDebugViewer.Common
 
 GETTEXT_DOMAIN = "gst-debug-viewer"
 
-def main_version (options):
+def main_version ():
 
     from GstDebugViewer import version
 
-- 
